scylla/src/statement/prepared_statement.rs

```diff
@@ -22,13 +22,18 @@ pub struct PreparedStatement {
 }
 
 impl PreparedStatement {
-    pub fn new(id: Bytes, metadata: PreparedMetadata, statement: String) -> Self {
+    pub fn new(
+        id: Bytes,
+        metadata: PreparedMetadata,
+        statement: String,
+        page_size: Option<i32>,
+    ) -> Self {
```

Comment:
Instead of an `Option` parameter, maybe we could add a builder-like `with_page_size` method, too?